bot.py

```diff
@@ -264,18 +382,6 @@ def riyans_denial(response):
             return True
     return False
 
-
-def new_user_message(response):
```

Comment:
Again a little confused on why new_user_message is being touched in an analytics feature branch